larpandoracontent/LArMonitoring/HierarchyValidationAlgorithm.cc

```diff
@@ -135,20 +135,40 @@ void HierarchyValidationAlgorithm::EventValidation(const LArHierarchyHelper::Mat
             if (primaries.size() == 0)
                 continue;
             primaries.sort(LArMCParticleHelper::SortByMomentum);
-            const InteractionDescriptor descriptor{LArInteractionTypeHelper::GetInteractionDescriptor(primaries)};
-
-            const int isCC{descriptor.IsCC()};
-            const int isQE{descriptor.IsQE()};
-            const int isResonant{descriptor.IsResonant()};
-            const int isDIS{descriptor.IsDIS()};
-            const int isCoherent{descriptor.IsCoherent()};
-            const int isNuMu{descriptor.IsMuonNeutrino()};
-            const int isNuE{descriptor.IsElectronNeutrino()};
-            const int nPiZero{static_cast<int>(descriptor.GetNumPiZero())};
-            const int nPiPlus{static_cast<int>(descriptor.GetNumPiPlus())};
-            const int nPiMinus{static_cast<int>(descriptor.GetNumPiMinus())};
-            const int nPhotons{static_cast<int>(descriptor.GetNumPhotons())};
-            const int nProtons{static_cast<int>(descriptor.GetNumProtons())};
+	    
+	    int isCC{0};
+            int isQE{0};
+            int isResonant{0};
+            int isDIS{0};
+            int isCoherent{0};
+            int isNuMu{0};
+            int isNuE{0};
+            int nPiZero{0};
+            int nPiPlus{0};
+            int nPiMinus{0};
+            int nPhotons{0};
+            int nProtons{0};
```

Comment:
I'd be inclined to define these on one line given they are all integers.

Reply:
Corrected